modules/theme-tools/compat/twentytwenty.css

```diff
@@ -0,0 +1,130 @@
+/**
+ * Infinite scroll
+ */
+
+/* Globally hidden elements when Infinite Scroll is supported and in use. */
+.infinite-scroll .pagination-wrapper,
+.infinite-scroll.neverending .footer-nav-widgets-wrapper,
+.infinite-scroll.neverending #site-footer {
+	/* Theme Footer (when set to scrolling) */
+	display: none;
+}
+
+ /* When Infinite Scroll has reached its end we need to re-display elements that were hidden (via .neverending) before. */
+.infinite-end.neverending .footer-nav-widgets-wrapper,
+.infinity-end.neverending #site-footer {
+	display: block;
+}
+
+.infinite-loader {
+	margin: calc(3 * 1rem) auto;
+}
+
+.infinite-loader .spinner {
+	margin: 0 auto;
+	left: inherit !important;
```

Comment:
Why the important? Could we increase the specificity of the selector?

Reply:
That's a good question. I think we need it since there are styles added inline directly:

![image](https://user-images.githubusercontent.com/426388/67502052-99a01800-f685-11e9-9ea3-a3df08e28bed.png)

Reply:
Ah. Makes sense.